### PR TITLE
Deprecate chunksize from dask_cudf.read_csv

### PR DESCRIPTION
**Pull request by rjzamora**
## Description
Deprecates `chunksize` in favor of `blocksize` to be consistent with `dask.dataframe.read_csv`.

Closes https://github.com/rapidsai/cudf/issues/11847

## Checklist
- [x] I am familiar with the [Contributing Guidelines](https://github.com/rapidsai/cudf/blob/HEAD/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.

### CODE REVIEW

**Review by galipremsagar — APPROVED** [review]

File: python/dask_cudf/dask_cudf/io/csv.py [hunk 1]
```diff
@@ -62,16 +62,32 @@ def read_csv(path, chunksize="256 MiB", **kwargs):
...
-        return _internal_read_csv(path=path, chunksize=chunksize, **kwargs)
+        return _internal_read_csv(path=path, blocksize=blocksize, **kwargs)
 
 
-def _internal_read_csv(path, chunksize="256 MiB", **kwargs):
-    if isinstance(chunksize, str):
-        chunksize = parse_bytes(chunksize)
+def _internal_read_csv(path, blocksize="256 MiB", **kwargs):
+    if isinstance(blocksize, str):
+        blocksize = parse_bytes(blocksize)
```
Comment (wence-) [inline python/dask_cudf/dask_cudf/io/csv.py:90]: `parse_bytes` already handles the case that its argument is an `int`

**Review by wence- — APPROVED** [review]
Looks good.

File: python/dask_cudf/dask_cudf/io/csv.py (hunk 1 above)
Comment (rjzamora) [reply python/dask_cudf/dask_cudf/io/csv.py]: oops - good point

**Comment by rjzamora** [comment]
@gpucibot merge

**Pull request merged**